AlphaWallet/AppCoordinator.swift

```diff
@@ -413,6 +413,12 @@ extension AppCoordinator: InCoordinatorDelegate {
 }
 
 extension AppCoordinator: UniversalLinkCoordinatorDelegate {
+
+    func handle(specialUrl url: URL, server: RPCServer, in coordinator: UniversalLinkCoordinator) {
```

Comment:
Can you rename the few occurrences of "specialUrl" to "embeddedUrl" or something like that?